composer.json

```diff
@@ -72,6 +71,9 @@
             "Neos\\Diff\\": [
                 "Neos.Diff/Classes"
             ],
+            "Neos\\Fusion\\Afx\\": [
+                "Neos.Fusion.Afx/Classes/"
```

Comment:
Probably irrelevant but all other packages do use the classes path without trailing slash?

Reply:
Should not matter but I will adapt it